plugins/feat/feat-nvme.c

```diff
@@ -464,7 +480,86 @@ static int feat_temp_thresh(int argc, char **argv, struct command *cmd, struct p
 		err = temp_thresh_set(dev_fd(dev), fid, opts, &cfg);
 	else
 		err = feat_get(dev, fid, NVME_SET(cfg.tmpsel, FEAT_TT_TMPSEL) |
-			       NVME_SET(cfg.thsel, FEAT_TT_THSEL), cfg.sel, timestamp_feat);
+			       NVME_SET(cfg.thsel, FEAT_TT_THSEL), cfg.sel, temp_thresh_feat);
+
+	return err;
+}
+
+static int arbitration_set(int fd, const __u8 fid, struct argconfig_commandline_options *opts,
+			   struct arbitration_config *cfg)
+{
+	__u32 result;
+	int err;
+	enum nvme_get_features_sel sel = NVME_GET_FEATURES_SEL_CURRENT;
+	__u8 ab;
+	__u8 lpw;
+	__u8 mpw;
+	__u8 hpw;
```

Comment:
just have them in one line.

also reorder deceleration to look more like in inverse christmas tree. E.g

```
enum nvme_get_features_sel sel = NVME_GET_FEATURES_SEL_CURRENT;
bool save = argconfig_parse_seen(opts, "save");
__u8 ab, lpw, mpw, hpw;
__u32 result;
int err;


```